railties/lib/rails/generators/rails/app/templates/config/puma.rb

```diff
@@ -0,0 +1,9 @@
+threads_count = ENV.fetch("MAX_THREADS") { 5 }.to_i
+threads threads_count, threads_count
+
+port        ENV.fetch("PORT") { 3000 }
+environment ENV.fetch("RAILS_ENV") { "development" }
+
+on_worker_boot do
```

Comment:
Connection/disconnection to Postgres/Redis/whatever adapter will automatically be handled on server boot. Do we want to add a specific disconnect method? Wouldn't be that hard to add. cc @matthewd